transfers/pre-transfer/fix_unlv_checksums.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import sys
+import os
+
+
+HASHFUNC_EXT = 'md5'
+
+
+def find_objects(path, ignore_dirs=None):
```

Comment:
Are you familiar with [`os.walk`](https://docs.python.org/2/library/os.html#os.walk)?  This looks like a reimplementation of it.